bundles/AdminBundle/Controller/Admin/MiscController.php

```diff
@@ -93,12 +94,17 @@ public function getAvailableControllersAction(Request $request, ControllerDataPr
      */
     public function getAvailableActionsAction(Request $request, ControllerDataProvider $provider)
     {
+        $routingDefaults = $this->getParameter('pimcore.routing.defaults');
         $bundle = $request->get('moduleName');
         if (empty($bundle)) {
-            $bundle = 'AppBundle';
+            $bundle = $routingDefaults['bundle'];
         }
 
         $controller = $request->get('controllerName');
+        if (empty($controller)) {
+            $bundle = $routingDefaults['controller'];
```

Comment:
This should be `$controller = ...` i guess.

Reply:
Of course, I fixed it